pool_automation/roles/plugins/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Execute role {{ plugins_role }}
+  include_role:
+    name: "{{ plugins_role }}"
+  when: not not plugins_role
```

Comment:
Why double `not`?

Reply:
for string->boolean coercion. without that you'll get something like `'plugins_role' is undefined`

Reply:
updated